dfastmi/io/Branch.py

```diff
@@ -0,0 +1,14 @@
+from dfastmi.io.Reach import Reach
+
+
+from typing import List
+
+
+class Branch():
+    name : str
+    qlocation : str
+    reaches : List[Reach]
```

Comment:
Maybe a dictionary of reaches based on reachname?
Or a object which we can add functionality to to find ceirtain reaches?
List can also be fine, just ideas :slightly_smiling_face::thought_balloon:

Reply:
No, no dictionaries. We want to use objects for object representations

Reply:
searching functionality only when needed. I don't see usage yet inscope of this issue.

Reply:
this comment might be more linked to how it is currently used by batch and gui, lets discuss at a later date.

Reply:
Discussed, we will leave these comments for a later refactoring.
This will be tackled in https://issuetracker.deltares.nl/browse/DFAST-112